examples/elasticsearch/src/test/kotlin/cnt/elasticsearch/BookServiceTest.kt

```diff
@@ -0,0 +1,150 @@
+package cnt.elasticsearch
+
+import cnt.elasticsearch.ElasticSearchConsts.ANALYSE_API
+import cnt.elasticsearch.ElasticSearchConsts.INDEX_API
+import cnt.elasticsearch.ElasticSearchConsts.INDEX_NAME
+import io.quarkus.test.junit.QuarkusTest
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import org.apache.http.util.EntityUtils
+import org.assertj.core.api.Assertions.assertThat
+import org.elasticsearch.client.Request
+import org.elasticsearch.client.Response
+import org.elasticsearch.client.RestClient
+import org.junit.jupiter.api.MethodOrderer
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestMethodOrder
+import javax.inject.Inject
+
+@QuarkusTest
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+class BookServiceTest {
+
+    @Inject
+    private lateinit var bookService: BookService
```

Comment:
Does the `QuarkusTest` extension support constructor injection?
Something like this?:
```
class BookServiceTest(
  @Inject private val bookService: BookService,
  @Inject private val elasticSearchClient: RestClient
) {
  ...
}
```

Reply:
Might be a personal preference, but I like to use `lateinit` and especially field injections only if absolutely needed.

Reply:
ok, sad ..